File: Models/Entities/UserDailyData.cs
using DAWProject.Models.Base;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace DAWProject.Models.Entities
{
	public class UserDailyData : BaseEntity
	{
		public DateTime Day { get; set; }
		public Double Weigth { get; set; }
		public Double BPM { get; set; }
		public int UserId { get; set; }
		public virtual User User { get; set; } 
	}
}

File: ClientApp/src/app/models/userdailydata.ts
import { User } from "./user";

export class UserDailyData {

  id: number;
  day: Date;
  weigth: number;
  bpm: number;
  userid: number;
  user: User;

  constructor(input?: any) {
    Object.assign(this, input);

  }

}
